packages/syft/src/syft/service/code/user_code.py

```diff
@@ -1285,26 +1294,48 @@ def generate_unique_func_name(context: TransformContext) -> TransformContext:
     return context
 
 
-def process_code(
-    context: TransformContext,
+def _check_global(code_tree: ast.Module) -> GlobalsVisitor | SyftWarning:
+    """
+    Check that the code does not contain any global variables
+    """
+    v = GlobalsVisitor()
+    try:
+        v.visit(code_tree)
+    except Exception:
+        raise SyftException(
+            "Your code contains (a) global variable(s), which is not allowed"
+        )
+    return v
+
+
+def _parse_code(raw_code: str) -> ast.Module | SyftWarning:
```

Comment:
Remove `_` from function name. Let's move `parse_code` and `check_for_global_vars` to `service/code/utils.py`

Reply:
Thanks for the review @shubham312. Done!